### DISCUSSION TITLE
E-mail on *.github.io domain

### DISCUSSION THREAD

**Discussion by Nepose** [community/community#22575] [Other Feature Feedback, Questions, & Ideas]
<p>Hello!</p>
<p>Is there any way to set up e-mail at GitHub pages? As an example, I have domain <a href="http://nepose.github.io" rel="nofollow noopener">nepose.github.io</a> and I want to add e-mail <a href="mailto:contact@nepose.github.io">contact@nepose.github.io</a>. I’d prefer to use this domain instead custom.</p>
<p>Thanks in advance :slight_smile:<br>
Nepose</p>

**Comment by jhvanderschee** [comment, marked as answer]
<p>No, that’s not possible.</p>
